pkg/resources/statefulsets/statefulset.go

```diff
@@ -192,7 +192,7 @@ func mysqlServerContainer(cluster *api.MySQLCluster, mysqlServerImage string, ro
 		"--innodb-print-all-deadlocks=ON",
 		"--innodb-undo-log-truncate=ON",
 		"--innodb-undo-tablespaces=2",
-		"--innodb-undo-logs=2",
+		// "--innodb-undo-logs=2", (removed in 8.0)
```

Comment:
@neumayer It would be great to get a review of these flags generally from you or one of your colleuges if possible as they were put together by Matt Lord for 5.7.20-1.1.2.

Reply:
Maybe just remove it